### PR TITLE
[ASV-59] Events that need to be fixed

### PR DESCRIPTION
**Pull request by JDSM01**
**What does this PR do?**

Makes the events on this ticket log with success https://aptoide.atlassian.net/browse/ASV-59
Account_Create_Your_Store_Screen - Event was not being logged and not called. Placed on ManageStorePresenter
Apps_Timeline_Follow_Friends - The place where the log was placed was unreachable. Moved to onViewCreatedClickOnAddressBook.
Stores_Tab_Open -  The string comparison could never be true due to wrong Enum used. Changed Enum to the correct.

**Database changed?**

No

**Where should the reviewer start?**

ManageStorePresenter.java

**How should this be manually tested?**

Try to reproduce the analytic events

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-13

**Questions:**

No libs

**Code Review Checklist**

- [X] Architecture
- [X] Documentation on public interfaces
- [ ] Database changed?
- [X] Remove comments & unused code & forgotten testing Logs
- [X] Codestyle
- [X] Partners build
- [X] Unit tests pass
- [ ] Functional QA tests pass

### CODE REVIEW

File: app/src/main/java/cm/aptoide/pt/account/view/store/ManageStoreFragment.java [hunk 1]
```diff
@@ -453,8 +459,8 @@ public void bindViews(View view) {
     storeImage = (ImageView) view.findViewById(R.id.create_store_image);
     storeName = (EditText) view.findViewById(R.id.create_store_name);
     storeDescription = (EditText) view.findViewById(R.id.edit_store_description);
-    cancelChangesButton = (Button) view.findViewById(R.id.create_store_skip);
-    saveDataButton = (Button) view.findViewById(R.id.create_store_action);
+    cancelChangesButton = (Button) view.findViewById(R.id.create_store_skip); //
+    saveDataButton = (Button) view.findViewById(R.id.create_store_action); //
```
Comment (phillipsgoncalves) [inline app/src/main/java/cm/aptoide/pt/account/view/store/ManageStoreFragment.java]: You forgot to remove these //

File: app/src/main/java/cm/aptoide/pt/account/view/store/ManageStorePresenter.java [hunk 2]
```diff
@@ -84,6 +91,12 @@ private Completable handleSaveClick(ManageStoreViewModel storeModel) {
         .observeOn(Schedulers.io())
         .andThen(saveData(storeModel))
         .observeOn(AndroidSchedulers.mainThread())
+        .doOnCompleted(() -> {
+          if (goBackToHome) {
+            accountAnalytics.createStore(view.hasImage(),
```
Comment (phillipsgoncalves) [inline app/src/main/java/cm/aptoide/pt/account/view/store/ManageStorePresenter.java]: Should use storeModel.hasPicture() instead

File: app/src/main/java/cm/aptoide/pt/account/view/store/ManageStorePresenter.java [hunk 3]
```diff
@@ -61,6 +64,10 @@ private void handleCancel() {
         .filter(event -> event == View.LifecycleEvent.CREATE)
         .flatMap(__ -> view.cancelClick()
             .doOnNext(__2 -> {
+              if (goBackToHome) {
+                accountAnalytics.createStore(view.hasImage(),
```
Comment (phillipsgoncalves) [inline app/src/main/java/cm/aptoide/pt/account/view/store/ManageStorePresenter.java]: Should pass the "empty" storeModel and get the picture from it instead

**Comment by analaragomes** [comment]
Forget this PR, look at this one:
https://github.com/Aptoide/aptoide-client-v8/pull/435

**Comment by analaragomes** [comment]
this was done one the other PR #435

**Pull request closed without merging**